Home work4

EgoVas95 opened the pull request.

Commit:
homework 4. Реализация ДЗ

Commit:
homework 4.  Почистил импорты

vlad-iv commented on hw03-spring-boot/src/test/resources/error.test.file.csv:
```diff
@@ -0,0 +1,2 @@
+#
```
Осталось добавить 
application.yml
и отключить в нем CommandLineRunner
command.line.runner.enabled=false

vlad-iv commented on hw03-spring-boot/pom.xml:
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Предлагаю смержить ДЗ-3, 1 и 2
Что бы в ПР было только ДЗ-4

Commit:
Merge branch 'main' into homeWork4

Commit:
homework 4.  Осталось добавить
             application.yml
             и отключить в нем CommandLineRunner
             command.line.runner.enabled=false

Commit:
homework 4. Изменил команду отключающую command.line.runner

Commit:
homework 4. Изменил команду отключающую command.line.runner

Commit:
homework 3. Отключил CommandLineRunner через профили

Commit:
homework 3. Отключил CommandLineRunner через профили

Commit:
homework 3. Убрал лишние импорты

Commit:
homework 4. downgrade java to 17v
